src/render/dom/HTMLVisualElement.ts

```diff
@@ -485,13 +483,20 @@ export class HTMLVisualElement<
 
         /**
          * If we have a listener for the viewport box, fire it.
-         * TODO: Instead of manually checking this, use framesync postRender
          */
-        if (!isReactRender) {
-            this.hasViewportBoxUpdated &&
-                this.config.onViewportBoxUpdate?.(this.targetBox, this.delta)
-            this.hasViewportBoxUpdated = false
-        }
+        this.hasViewportBoxUpdated &&
```

Comment:
This function is no longer being triggered from build so it won't fire synchronously with a React render